pkg/util/parsing_helper_test.go

```diff
@@ -92,3 +92,9 @@ func TestParseMinusSuffix(t *testing.T) {
 	assert.DeepEqual(t, expectedMap, inputMap)
 	assert.DeepEqual(t, expectedStringToRemove, stringToRemove)
 }
+
+func TestAddListAndRemovalListFromArray(t *testing.T) {
```

Comment:
I would add tests for corner cases. For instance, 
1. add one item and remove none, 
2. add no items, remove one, 
3. add one item and remove one, 
4. add none and remove none.

Reply:
will do.

Reply:
done.